packages/communication-react/tsconfig.json

```diff
@@ -4,13 +4,13 @@
     "baseUrl": "./src",
     "outDir": "./dist/dist-esm",
     "paths": {
-      "@internal/chat-component-bindings": ["../../chat-component-bindings/src"],
-      "@internal/calling-component-bindings": ["../../calling-component-bindings/src"],
-      "@internal/acs-ui-common": ["../../acs-ui-common/src"],
-      "@internal/calling-stateful-client": ["../../calling-stateful-client/src"],
-      "@internal/chat-stateful-client": ["../../chat-stateful-client/src"],
-      "@internal/react-components": ["../../react-components/src"],
-      "@internal/react-composites": ["../../react-composites/src"]
+      "@internal/chat-component-bindings": ["../../chat-component-bindings/src/index"],
+      "@internal/calling-component-bindings": ["../../calling-component-bindings/src/index-public"],
```

Comment:
this one should just be `index` not `index-public`

Reply:
Why do you say that?

Reply:
Ah yep `@internal/calling-stateful-client` has the public index not the `@internal/calling-component-bindings`

Reply:
Updated

Reply:
OK, reverted that packlet back to just `index` and not `index-public`, there's not the correct re-exports in the meta package -- would need to be its own PR